src/components/FAQ/QAItem/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const QAItem = ({ question, answer }) => {
+  return (
+    <ul className="dp-accordion">
+      <li>
+        <span className="dp-accordion-thumb">
+          <b>{question}</b>
+        </span>
+        <div className="dp-accordion-panel">
+          <div className="dp-accordion-content">{answer}</div>
+        </div>
```

Comment:
What do you think about using MarkDown for the questions and answers? It will allow using rich text capabilities that could be useful (italic, bold, paragraphs, etc)

Reply:
Oh, it is already MarkDown: https://github.com/FromDoppler/doppler-webapp/pull/1443/files#diff-b31493918b33c77f68d48486906c5edc47d0935b59f7e6f0d64635bf46faabffR27